examples/site/howto_offload_builder_creation/README.md

```diff
@@ -0,0 +1,148 @@
+# How-to Guide: Use Cloud Build to create a buildpacks builder
+
+[repository-gh]: https://github.com/GoogleCloudPlatform/functions-framework-cpp
+[cloud-build-quickstarts]: https://cloud.google.com/build/docs/quickstarts
+[gcp-quickstarts]: https://cloud.google.com/gcp/getting-started
+[buildpacks]: https://buildpacks.io
+[docker]: https://docker.com/
+[docker-install]: https://store.docker.com/search?type=edition&offering=community
+[sudoless docker]: https://docs.docker.com/engine/install/linux-postinstall/
+[pack-install]: https://buildpacks.io/docs/install-pack/
+[hello-world-http]: /examples/site/hello_world_http/hello_world_http.cc
+[cloud-build]: https://cloud.google.com/cloud-build
+
+This guide shows you how to use [Cloud Build][cloud-build] to offload the
+creation of [buildpacks] builders.
+
+## Pre-requisites
+
+This guide assumes you are familiar with Google Cloud, and that you have a GCP
+project with Cloud Build enabled. If needed, consult:
+* the [GCP quickstarts][gcp-quickstarts] to setup a GCP project
+* the [cloud build quickstarts][cloud-build-quickstarts] to setup Cloud Build
+  in your project
+
+This guide shows you how to test the builder by using it to create a local
+container. These steps will require you to have [docker] and the
+[pack tool][pack-install] on your workstation.
+
+Verify the [docker tool][docker] is functional on your workstation:
+
+```shell
+docker run hello-world
+# Output: Hello from Docker! and then some more informational messages.
+```
+
+If needed, use the [online instructions][docker-install] to download and install
+this tool. This guide assumes that you have configured [sudoless docker], if
+you prefer replace all `docker` commands below with `sudo docker`.
+
+Verify the [pack tool][pack-install] is functional on our workstation. These
+instructions were tested with v0.17.0, they should work with newer versions.
+Some commands may not work with older versions.
+
+```shell
+pack version
+# Output: a version number, e.g., 0.17.0+git-d9cb4e7.build-2045
+```
+
+## Getting the code for this example
+
+This example is included in the Functions Framework for C++
+[source code repository][repository-gh]. Download this code as usual:
+
+```shell
+cd $HOME
+git clone https://github.com/GoogleCloudPlatform/functions-framework-cpp
+```
+
+The rest of this guide will assume you are issuing commands in the framework's
+clone:
+
+```shell
+cd $HOME/functions-framework-cpp
+```
+
+## Create the buildpacks builder
+
+Set the `GOOGLE_CLOUD_PROJECT` shell variable to the project id of your GCP
+project, and schedule a build to create your buildpack:
+
+```shell
+GOOGLE_CLOUD_PROJECT=... # put the right value here
+gcloud builds submit \
+    "--project=${GOOGLE_CLOUD_PROJECT}" \
```

Comment:
> It's usually best to `gcloud config set project` rather than use `--project`.

I find that worse. It makes it hard to run separate instructions in separate terminals, and it breaks in terrible ways if you forget the `gcloud config set` steps.  Stateless commands are easier to re-run, and fail with clearer error messages. But I do realize that is maybe an uncommon opinion, and there is value in consistency for these things.  I created #268 to clean all the README files.

> 
> Can we change to the script to not use `--project`?

The script works with `gcloud config set project` too.  You mean the README?